R/variantSetTests.R

```diff
@@ -55,14 +55,15 @@ testVariantSet <- function( nullmod, G, weights,
     
     # adjust burden for covariates and random effects
     Gtilde <- calcGtilde(nullmod, burden)
+    RSS0 <- as.numeric(crossprod(nullmod$Ytilde))
     
     if (burden.test == "Score") {
-        out <- .testGenoSingleVarScore(Gtilde, G = burden, resid = nullmod$resid) 
-    }
-    if (burden.test == "Wald"){
-        out <- .testGenoSingleVarWald(Gtilde, Ytilde = nullmod$Ytilde,
-                                      n = length(nullmod$Ytilde), k = ncol(nullmod$model.matrix))
+        out <- .testGenoSingleVarScore(Gtilde, G = burden, resid = nullmod$resid, RSS0 = RSS0) 
```

Comment:
and the if statement here could just go away, right?

Reply:
When we add the carrier tests, there will still be if statements here:
`if (burden.test == "BinomiRare") out <- .testGenoSingleVarBR()`